templates/error.html

```diff
@@ -1,11 +1,17 @@
 <!DOCTYPE html>
 <html>
 <head>
-	<title>Your request returned an error</title>
+	<meta charset="UTF-8">
+	<title>Albums</title>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-QWTKZyjpPEjISv5WaRU9OFeRpok6YctnYmDr5pNlyT2bRjXh0JMhjY6hW+ALEwIH" crossorigin="anonymous">
 </head>
-<body>
-	<h1>Your request returned an error</h1>
-	<p>{{.Error}}</p>
+<body class="pt-3">
+	<div class="container">
+		<h1>Your request returned an error</h1>
+		<p>{{.Error}}</p>
+	</div>
 </body>
+{{ template "footer" . }}
 </body>
```

Comment:
This doesn't lookk right, but I'll fix the UI stuff next. No need to do it here again.